templates/vue-demo-store/components/account/AccountRegisterForm.vue

```diff
@@ -109,14 +109,13 @@ const invokeSubmit = async () => {
     try {
       loading.value = true;
       const response = await register(state);
-      if (response && response.active) router.push("/");
-      else if (response && !response.active) {
+      if (response && response.doubleOptInRegistration) {
         Object.assign(state, JSON.parse(JSON.stringify(initialState)));
         showDoubleOptInBox.value = true;
         await nextTick();
         doubleOptInBox.value?.scrollIntoView();
         $v.value.$reset();
-      }
+      } else if (response && response.active) router.push("/");
```

Comment:
looks like there is no info for user with that flow, they are neither lonned in or have info to check the email, right?

Reply:
cool, but one more thing - after the registration the form should not be visible anymore, just the notification
![image](https://github.com/user-attachments/assets/0d1ac633-6d23-4e94-874a-151e4856e6f4)

Reply:
This is the same flow we have on the storefront. We decided to keep them as similar as possible

Reply:
done

Reply:
I created a task to format this page https://github.com/shopware/frontends/issues/1496